controllers/projectsController.js

```diff
@@ -0,0 +1,45 @@
+var db = require('../models');
+
+// GET /api/projects
+function index(req, res) {
+  // access database and pull out all projects
+  db.Album.find({}, function(err, allAlbums) {
```

Comment:
Replace Album with the correct Schema name

Reply:
+1